2023/11.md

```diff
@@ -24,7 +24,7 @@ gantt
 For meeting times in your timezone, visit [Temporal docs](https://tc39.es/proposal-temporal/docs/) and run the code below in the devtools console.
 
 ```js
-Temporal.ZonedDateTime.from('2023-11-17T10:00[America/Los_Angeles]')
```

Comment:
As written, this gives you the advancement deadline. I think that's fine.

Reply:
That would be fine if it said "for _the advancement deadline_ in your timezone", but it claims to be the meeting time.